chapters/en/slides/module3/module3_02.md

```diff
@@ -137,59 +149,86 @@ Notes: **The green “Code” drop-down menu contains the remote address (URL) c
 
 <img src='/module3/vc-clone-jupyter-2.png' width="60%" alt="404 image"/>
 
-<center/>
+</center>
+
+Notes:
+Next, open JupyterLab and click the Jupyter Git Clone icon in the "Files" tab of the left side bar. We have marked this icon with a red circle in the screenshot in this slide. When you click this button a dialog box will show up, which we will explain in the next slide.
```

Comment:
We should add semantic break lines to these paragraphs, I can do it later if the rest of the course is going to have them

Reply:
Yes ideally we would have semantic line breaks everywhere. I didn't bother doing it for this review since it would mark everything as changed even though I didn't change the content, but just the formatting. Feel free to do it later if you have time, but don't worry too much about it =)